extensions/libraries/redcore/model/list.php

```diff
@@ -454,6 +454,9 @@ protected function loadForm($name, $source = null, $options = array(), $clear =
 			// We pass the data because plugins may require it.
 			$this->preprocessForm($form, $data);
 
+			// Filter and validate the form data.
+			$data = $form->filter($data);
```

Comment:
@Kixo @phproberto @jatitoam please take a look at this string. I'd prefer to use data filter for filter forms also. It doesn't use in joomla core. This is data filter will help to convert datetime fields in the right UTC offset.
What do you think about it?